guides/introduction/consuming.md

```diff
@@ -1,27 +1,123 @@
 # Consuming media data
 
-Other than just forwarding, we probably would like to be able to use the media right in the Elixir app to
-e..g feed it to a machine learning model or create a recording of a meeting.
+Other than just forwarding, we would like to be able to use the media right in the Elixir app to e.g.
+use it as a machine learning model input, or create a recording of a meeting.
 
-In this tutorial, we are going to build on top of the simple app from the previous tutorial by, instead of just sending the packets back, depayloading and decoding
-the media, using a machine learning model to somehow augment the video, encode and payload it back into RTP packets and only then send it to the web browser.
+In this tutorial, we are going to depayload and decode received video data to use it for ML inference.
 
-## Deplayloading RTP
+## Depayloading RTP
 
-We refer to the process of taking the media payload out of RTP packets as _depayloading_.
+We refer to the process of getting the media payload out of RTP packets as _depayloading_. It may seem straightforward at first,
+we just take the payload of the packets and we get a stream of media data. Sometimes it is that simple, like in the
+case of Opus-encoded audio, where each of the RTP packets is, more or less, 20 milliseconds of audio, and that's it.
 
 > #### Codecs {: .info}
```

Comment:
What about inlining this and changing the order of the whole section a little bit?
Maybe let's start with explaining what browser does i.e.

```
When browser sends audio or video it does the following steps:
* aquire audio or video frame
* encode it so it takes less space and ueses less bandwidth 
* depending on its size, pack into a single or multiple RTP packets 

When we receive audio or video from a web browser, we have to do the opposite:
* collect multiple RTP packets that contain a single audio or a single video frame
* unpack this frame from RTP packets
* decode it

In practice, audio frames always fit into a single RTP packet.
In case of video, this depends on its resolution and compression level.
However, you don't have to worry about this. 
Just feed RTP packets into so called depayloader and it will return you a frame once it collect all needed data.
```